src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
do not reset margin and padding for all selectors. only for elements which necessary
 ```suggestion
```